api/DHSC.FingertipsNext.Modules.HealthData.Service/IndicatorService.cs

```diff
@@ -137,7 +149,7 @@ IndicatorPolarity polarity
         }
 
         //if RAG is the benchmark method use England as the comparison area and add England to the areas we want data for
-        var benchmarkAreaCode = AreaCodeEngland; //for PoC all benchmarking is against England, post PoC this will be passed in as a variable
+        var benchmarkAreaCode = benchmarkRefType == BenchmarkReferenceType.AreaGroup ? areaGroup: AreaCodeEngland; //for PoC all benchmarking is against England, post PoC this will be passed in as a variable
```

Comment:
Does the comment need updating to say use the areaGroup if the BenchmarkReferenceType is now AreaGroup. Actually do we need the top comment at all anymore?